src/pages/Dashboard.jsx

```diff
@@ -0,0 +1,20 @@
+import AdminPollList from "../components/admin/AdminPollList";
+import UserPoll from "../components/users/UserPoll";
+import { jwtDecode } from "jwt-decode";
+import { useNavigate } from "react-router-dom";
+
+const Dashboard = () => {
+  const showUser = localStorage.getItem("polltoken");
+  const decoded = jwtDecode(showUser);
+  const navigate = useNavigate();
+  if (!showUser) {
+    navigate("/");
+  }
```

Comment:
By this logic, I can access other routes if even I have not logged-In yet.
Did I ask you to implement it?